src/maus/models/edifact_components.py

```diff
@@ -97,6 +98,46 @@ def deserialize(self, data, **kwargs) -> DataElementFreeText:
         return DataElementFreeText(**data)
 
 
+@attrs.define(auto_attribs=True, kw_only=True)
+class ValuePoolEntry:
+    """
+    A value pool entry contains the EDIFACT qualifier, a meaning (German text) and an ahb expression.
+    A value pool consists of 1 to n ValuePoolEntries.
+    The data element 3055 in UTILMD is a good example for a value pool.
+    It is used in the segments NAD+MS and NAD+MR. Its ValuePoolEntries are:
+    - (key: "9", meaning: "GS1", ahb_expression: "X")
+    - (key: "293", meaning: "DE, BDEW", ahb_expression: "X")
+    - (key: "332", meaning: "DE, DVGW", ahb_expression: "X")
+    """
+
+    #: the qualifier in edifact, might be f.e. "E01", "D", "9", "1.1a", "G_0057"
+    edifact_key: str = attr.field(validator=attrs.validators.matches_re(r"^[A-Z\d\.a-z_]+$"))
+    #: the meaning as it is written in the AHB (f.e. "Einzug", "Entwurfs-Version", "GS1", "Codeliste Gas G_0057"
+    meaning: str = attr.field(validator=attrs.validators.instance_of(str))
+    #: the ahb expression, in most cases this is a simple "X"; it must not be empty
+    ahb_expression: str = attr.field(validator=attrs.validators.matches_re(".+"))
+    # must not be empty (if so, the value pool entry should not be included of the result)
+
+
+class ValuePoolEntrySchema(Schema):
+    """
+    A Schema to serialize ValuePoolEntries
+    """
+
+    # this looks like a plain Dict[str,str] but we prefer typed access over loose string key value pairs
+    edifact_key = fields.String(required=True)
+    meaning = fields.String(required=True)
+    ahb_expression = fields.String(required=True)
+
+    # pylint:disable=unused-argument,no-self-use
+    @post_load
+    def deserialize(self, data, **kwargs) -> ValuePoolEntry:
+        """
+        Converts the barely typed data dictionary into an actual :class:`.ValuePoolEntry`
+        """
+        return ValuePoolEntry(**data)
```

Comment:
I thought we have a better method to deserialise. Just by defining the class name somewhere

Reply:
this was in bo4e, where we referenced the model class separatly